api/app.py

```diff
@@ -376,10 +376,36 @@ def derive_object(df):
                      "source":df.iloc[0]["SourceUrl"],
                      "taxonomy":df.iloc[0]["Taxonomy"]}
         out       = {**figures, **text_data}
-        return out
+
+        return pd.Series(out)
+
     gg = relevant_rows.groupby(["Fact","Taxonomy", "SourceUrl"], as_index=False)
-    out = gg.apply(derive_object).values.tolist()
-    return out
+
+    out = gg.apply(derive_object)
+
+    translate_dict = {"Fact": "summary", "SourceUrl":"source", "Taxonomy":"taxonomy"}
+    passthru = ["agree", "disagree", "discuss"]
+
+    out_list = []
+    items = out.values.tolist()
+    for i in items:
+        it = dict(zip(out.columns, i))
+        d = {translate_dict[k]: v for k, v in it.items() if k in translate_dict}
+        for c in passthru:
+            d[c] = it[c]
+        out_list.append(d)
+
+    return out_list
+
+@app.route('/api/v1/mtweets') # endpoint for user to query for tweet view
```

Comment:
Needs to parse the files similar to `2020-05-01_to_2020-05-10_ennonrt_0_stnc_dtctn.csv` to return tweet text as that file contains the tweet text in same row as the fact we will receive from javascript.

```shell
➜  processed head -n 2 2020-05-01_to_2020-05-10_ennonrt_0_stnc_dtctn.csv
Timestamp,TweetText,Fact,Details,Taxonomy,SourceUrl,UserCountry,UserState,UserCounty,TweetCountry,TweetState,TweetCounty,similarity,agree,discuss,disagree,stance
2020-05-01 06:12:02+00:00,"“Big Brother” In Germany Hasn’t Told Contestants About The Coronavirus, So They’re Planning To Break The https://t.co/pM4bEzIjJ3",There is no proven cure for the coronavirus. ,"There is no proven cure for the coronavirus. But that has not stopped people (and some businesses) from claiming to have found one. 
```

So we can use the `TweetText,Fact` to populate the `TweetText` items from the `Fact` item.

In addition, we may need to store the entire `details` variable to match against the `Fact` variable since we are currently using the `summary`. Although it might be possible that the `summary` will suffice.

Reply:
Data being returned:

![Screen Shot 2021-01-21 at 12 50 59 AM](https://user-images.githubusercontent.com/19378168/105310282-c5f55d80-5b82-11eb-9d2a-aae5fcdfd947.png)